h2/stream.py

```diff
@@ -572,10 +572,9 @@ def locally_pushed(self):
         immediately after initialization. Sends no frames, simply updates the
         state machine.
         """
-        events = self.state_machine.process_input(
-            StreamInputs.SEND_PUSH_PROMISE
-        )
-        return [], events
+        # This does not trigger any events.
+        self.state_machine.process_input(StreamInputs.SEND_PUSH_PROMISE)
```

Comment:
For any call into the state machine, can we assert on the return value? This'll stop us getting this wrong or accidentally throwing away anything we need.